libdnf-cli/output/transaction_table.hpp

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF_CLI_OUTPUT_TRANSACTION_TABLE_HPP
+#define LIBDNF_CLI_OUTPUT_TRANSACTION_TABLE_HPP
+
+#include <fmt/format.h>
+#include <libsmartcols/libsmartcols.h>
+#include <vector>
+
+#include "../../libdnf-cli/utils/units.hpp"
+
+//static struct libscols_table * create_transaction_table(bool with_status) {}
+
+//static void add_line_into_transaction_table() {}
+
+//template <class Query>
+//static void print_transaction_table(Query query, bool with_status) {}
+
+namespace libdnf::cli::output {
+
+// transaction det
+enum { COL_NEVRA, COL_REPO, COL_SIZE };
```

Comment:
Prefer `enum class` over 'enum'. Also the comment line is sort of unclear / unfinished.